stl/inc/xloctime

```diff
@@ -666,6 +666,110 @@ _NODISCARD constexpr bool _Is_valid_strftime_specifier(const char _Specifier) {
     return false;
 }
 
+_NODISCARD constexpr bool _Is_valid_strftime_tm_sec(const tm* const _Pt) noexcept {
+    // seconds after the minute - [0, 60] including leap second
+    return _Pt->tm_sec >= 0 && _Pt->tm_sec <= 60;
+}
+
+_NODISCARD constexpr bool _Is_valid_strftime_tm_min(const tm* const _Pt) noexcept {
+    // minutes after the hour - [0, 59]
+    return _Pt->tm_min >= 0 && _Pt->tm_min <= 59;
+}

```

Comment:
I wanted to do these as internal lambdas, but on older standards this is not allowed and because this header is included as part of the public includes I have to make it compatible with all flavors.

Reply:
Does this stuff actually need to be `constexpr`? It's in iostreams, so the top-level is non-`constexpr`, unless I am confused.

Reply:
Well you did so for the previous PR where I check for invalid specifies, instead of using static local variable, so I thought we are seeking the same for this part. Otherwise I can move them to lambdas, no problem.

Reply:
Oh, that was basically a case of "`constexpr` doesn't hurt here, so I have no objections". Unlike applying `const` to variables and parameters whenever possible, we have no consistent convention about marking helper functions as `constexpr` when possible.